### ISSUE TITLE
Extracting usernames from matched line

### ISSUE THREAD

**Issue by pigmonkey** [klaasb/passdmenu#6]
In reply to your comment in the `get_user_pw()` function, I think it is probably better to split the line only at the first white space character.

Currently, if the username has a space in it, you're only returning the first "word" of it. You can use `maxsplit=1` to split at the first occurrence.

```
>>> 'login: john doe'.split()
['login:', 'john', 'doe']
>>> 'login: john doe'.split(maxsplit=1)
['login:', 'john doe']
```

[passff](https://github.com/jvenant/passff) expects the login (and url) field names to be separated by a colon. I like that personally because it encourages me to format everything after the password on line 1 as YAML, which helps me keep entries clean and parsable. If you wanted to follow that convention you could just split at the first `:` and then strip leading/trailing white space, but I don't know if that's really any more useful than splitting at the first white space.

**Comment by klaasb** [comment]
It does sound like a better specification to me and if we'd use something like YAML, this can make other use cases for extracting info and acting on it simpler.

**Comment by klaasb** [comment]
Solved by PR #9
